app/views/foreman_virt_who_configure/api/v2/configs/deploy_script.json.rabl

```diff
@@ -0,0 +1,3 @@
+object @config
+
+attributes :virt_who_config_script
```

Comment:
This response could also contain hash of possible error codes when we merge https://github.com/theforeman/foreman_virt_who_configure/pull/53